core-bundle/src/Resources/config/controller.yml

```diff
@@ -0,0 +1,12 @@
+services:
+    _defaults:
+        autoconfigure: true
+
+    _instanceof:
+        Symfony\Bundle\FrameworkBundle\Controller\AbstractController:
+            calls:
+                - [setContainer, ['@Psr\Container\ContainerInterface']]
+
+    Contao\CoreBundle\Controller\Page\RootPageController:
+        tags:
+            - { name: contao.page, contentComposition: false }
```

Comment:
We are using Annotations everywhere else now. Why explicitly tag it here? For consistency I would also use annotations for the page controllers and change this to `Contao\CoreBundle\Controller\Page\RootPageController: ~`

Reply:
good question. I'm not sure we have a consistency here, we just merged https://github.com/contao/contao/blob/4.x/core-bundle/src/Resources/config/services.yml#L111-L113 for example. But we seem to use it for event listeners (hooks & callbacks) a lot …

Reply:
Ah yeah, that went under my radar too.

Reply:
Changed in 5dd58a27a0e60ea8a1c26b5811bb9a3672390fbf
I'll do a follow-up PR once this is merged to clean it up everywhere